build.gradle

```diff
@@ -179,7 +179,7 @@ minecraft {
 
             property 'forge.logging.console.level', 'info'
 
-            args '--mod', 'minecolonies', '--all', '--output', file('src/main/generated/resources'), '--input', file('src/main/resources')
+            args '--mod', 'minecolonies', '--all', '--output', file('src/main/generated/resources'), '--input', file('src/main/resources'), '--existing', file('src/main/resources')
```

Comment:
I did say you could just replace `--input` with `--existing`; you don't need to duplicate the parameter and path.  But this is ok too.

Reply:
I didn't add the original `--input` argument, so I'm a little worried about making gradle modifications that remove anything unless absolutely necessary.

Reply:
I've tested that in another mod and it works perfectly fine.  But as I said, while this is unnecessary it's not harmful, so it's ok to stay like this if you prefer.